app/controllers/job_invocations_controller.rb

```diff
@@ -63,7 +63,7 @@ def show
     @job_invocation = resource_base.includes(:template_invocations => :run_host_job_task).find(params[:id])
     @job_organization = Taxonomy.find_by(id: @job_invocation.task.input[:current_organization_id])
     @job_location = Taxonomy.find_by(id: @job_invocation.task.input[:current_location_id])
-    @auto_refresh = @job_invocation.task.try(:pending?)
+    @auto_refresh = @job_invocation.task.try(:pending?) || has_pending_hosts?
```

Comment:
The `has_pending_host?` only comes into play if `@job_invocation.task.try(:pending?)` returns a false-y value. This can happen if either `@job_invocation.task` does not exist yet or if the task exists but is already done, which would imply all its children have already finished too.

If the `@job_invocation.task` does not exist yet, the child tasks cannot exist so `has_pending_hosts?` will be false as well.

If `@job_invocation.task.pending?` returns false, then all its children should already be done too, so `has_pending_hosts?` _should_ always be false too.